test/jdk/java/beans/Introspector/Test5102804.java

```diff
@@ -26,7 +26,7 @@
  * @bug 5102804
  * @summary Tests memory leak
  * @author Sergey Malenkov
```

Comment:
Could you also remove the `@author` tag?

Reply:
Done - I've now updated the PR to remove these author tags. Tests continue to pass.